recipes-connectivity/network-manager-applet/network-manager-applet/disable-show-password.patch

```diff
@@ -0,0 +1,39 @@
+--- network-manager-applet-0.9.2.0/src/wireless-security/wireless-security.c	2017-01-13 10:50:54.830968023 -0500
++++ network-manager-applet-0.9.2.0/src/wireless-security/wireless-security.c	2017-01-16 15:23:41.424841517 -0500
+@@ -37,6 +37,20 @@
+ #include "wireless-security.h"
+ #include "eap-method.h"
+ 
++static void
++removeCheckboxes (GtkBuilder *builder)
```

Comment:
Why not a patch that removes the checkbox creation code, instead of adding code to hide them?